Locale/enUS/config.lua

```diff
@@ -62,6 +62,8 @@ QuestieLocale.locale['enUS'] = {
     ['ENABLE_AUTO_ACCEPT_QUESTS_DESC'] = "Enable or disable Questie auto-accepting quests.",
     ['ENABLE_AUTO_COMPLETE'] = "Auto Complete",
     ['ENABLE_AUTO_COMPLETE_DESC'] = "Enable or disable Questie auto-complete quests.",
+    ['ENABLE_AUTO_DIALOG'] = "Enable Automatic NPC Dialog",
+    ['ENABLE_AUTO_DIALOG_DESC'] = "Enable or disable Questie automatically advancing through NPC dialog.",
```

Comment:
It would be great to copy paste these lines into all the other localization files. That is the current way to keep the translations in a shape which allows to find missing translations.

Reply:
Done: https://github.com/Questie/Questie/pull/2566/commits/cdd3d561ef9b7c178a466dce9d0fc4030644ad1a